code/ui/PostRoundMenu.scss

```diff
@@ -0,0 +1,32 @@
+PostRoundMenu {
+    background-color: #0009;
+    border: 1px solid #000;
+    border-radius: 8px;
+
+    max-width: 600px;
+    width: 100%;
+    left: 50%;
+    top: 35%;
+    transform: translateX(-50%) translateY(-50%);
+    position: absolute;
+    flex-direction: column;
+
+    color: white;
+    font-family: Poppins;
+    font-weight: 500;
+
+    &.hide {
+        display: none;
```

Comment:
Why not opacity:0... this is better in most cases because you can set transitions on opacity but not display:none